src/mixins/labels/labelsHolder.js

```diff
@@ -0,0 +1,23 @@
+import * as THREE from "three";
+
+export class LabelsHolder {
+    constructor(config) {
+        this.labelType = config.labelType || "labels";
```

Comment:
Should be at the class level, we can use Typescript

Reply:
Agree. Why do we skip TS definitions almost everywhere in this PR?